t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/p2p/discover"
-	"github.com/ethereum/go-ethereum/whisper/whisperv5"
+	"github.com/ethereum/go-ethereum/whisper/whisperv6"
```

Comment:
We can do a similar trick as we usually do, i.e. `whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"`.